README.md

```diff
@@ -107,9 +118,33 @@ auto-round-fast \
     --group_size 128 \
     --disable_eval 
   ```
+<br>
+
+#### Auto-Round Recipes Results
+In general, it is recommended to use the auto-round default mode. When resources or quantization time are a priority, the auto-round-light mode can be preferred for models larger than 3B. Below are the quantization results for models ranging from 3B to 72B as a reference(with torch compile enabled).
```

Comment:
Simply this line.  for 2bits, we recommend auto-round-best and better present 2bit results